packages/module/src/ConsoleInternal/components/catalog/_catalog.scss

```diff
@@ -115,7 +115,7 @@ $catalog-tile-width: $pfext-m-catalog-tile-width;
 
   &__btn-group__group-by {
     display: inline;
-    margin-left: var(--pf-v5-global--spacer--md);
+    margin-left: var(--pf-v6-global--spacer--md);
```

Comment:
for the most part with these t-shirt sized spacer vars, you can just `s/pf-v6-global--spacer/pf-t--global--spacer` to use the new token values